make.sh

```diff
@@ -7,7 +7,7 @@ USERGUIDE_INPUT_DIR=$DIR/userguide
 # Common options
 COMMON_OPTS="-a sourcedir=$USERGUIDE_INPUT_DIR -a imagesdir=$USERGUIDE_INPUT_DIR/assets"
 # Selfcontained options
-SELFCONTAINED_EXTRA_OPTS="$COMMON_OPTS -T html5 -a stylesheet=kb.css -a stylesdir=../stylesheets -a source-highlighter=pygments -a pygments-style=monokai"
+SELFCONTAINED_EXTRA_OPTS="$COMMON_OPTS -T html5 -a stylesheet=style.css -a stylesdir=../stylesheets -a source-highlighter=pygments -a pygments-style=monokai"
```

Comment:
This won't work without changing the template files:

![image](https://user-images.githubusercontent.com/51940/105673604-2c111600-5ede-11eb-927e-a6f27696cfca.png)

Reply:
@pierre , how can I access this page, in local?